src/Microsoft.Azure.WebJobs.Host/Executors/StorageAccountProviderExtensions.cs

```diff
@@ -82,16 +86,27 @@ private static void ValidateStorageAccount(IStorageAccount account, string conne
         }
 
         /// <summary>
-        /// Walk from the parameter up to the containing type, looking for a
-        /// <see cref="StorageAccountAttribute"/>. If found, return the account.
+        /// Determine whether the specified parameter declares an explicit storage
+        /// account to use, either by specifying a value for <see cref="StorageAttribute.Account"/> or
+        /// if <see cref="StorageAccountAttribute"/> has been applied.
         /// </summary>
         internal static string GetAccountOverrideOrNull(ParameterInfo parameter)
```

Comment:
Yes, this is still needed for CloudStorageAccount/CloudQueueClient type bindings, e.g. https://github.com/Azure/azure-webjobs-sdk/blob/dev/test/Microsoft.Azure.WebJobs.Host.EndToEndTests/MultipleStorageAccountsEndToEndTests.cs#L230. In such cases, there is no Queue attribute.